Cart/Cart/ItemManager.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Cart
+{
+    public class ItemManager
+    {
+        public void AddProduct(List<ICartItem> cartItemList, IProduct product, int quantity)
+        {
+            
+            bool cartItemFound = false;
+            foreach (ICartItem item in cartItemList)
```

Comment:
It is better to keep a function like IsCartItemPresent() which returns bool so that u can reuse it